app/components/dashboard/all_collections_row_component.rb

```diff
@@ -0,0 +1,32 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dashboard
+  # Renders a list of all collections
+  class AllCollectionsRowComponent < ApplicationComponent
+    sig { params(collection: Collection, counts: Hash).void }
+    def initialize(collection:, counts:)
+      @collection = collection
+      @counts = counts
+    end
+
+    attr_reader :collection, :counts
+
+    def draft?
+      return true if @collection.head&.state&.include? 'draft'
+
+      false
+    end
+
+    def draft_label
```

Comment:
why not return all the html from this method?

```ruby
  tag.strong do
   tag.i  ' - Draft'
  end
 ```

Reply:
better sill might be:
```ruby
tag.span ' - Draft', class: 'draft-tag'
```

and then add a `.draft-tag` css style.

Reply:
This is what I was looking for! Thanks, I also wasn't happy with the HTML that way.